server/plugin/command.go

```diff
@@ -17,6 +17,7 @@ const (
 	featureIssues        = "issues"
 	featurePulls         = "pulls"
 	featurePullsMerged   = "pulls_merged"
+	featurePullsCreated  = "pulls_new"
```

Comment:
To be consistent with the flag pertaining to new issues `issue_creations`, I'm thinking this should be named `pull_creations`. What do you think @C-Deck? Just an idea, not too sold on it

Reply:
@mickmister I honestly thought it should be `pulls_created` to match `pulls_merged`. I only went with `new_pulls` originally because that is what the ticket had. Any of the three are fine by me though.

Reply:
@hanzei @mickmister Should I change it? Not sure if this was a yes or not.

Reply:
`pulls_merged` sounds good to me.